modules/ROOT/pages/prerequisites/prerequisites.adoc

```diff
@@ -389,3 +389,20 @@ Ceph Notes::
 
 * Note that you cannot access the same files in CephFS and Ceph-S3. Ceph allows exposing commodity hardware as either blockstorage (RBD), S3 or CephFS. It is not possible to write a file via S3 and then read it via CephFS.
 ====
+
+== Compatible Clients
+
+At the time of release, users will be able to enjoy the spaces feature of Infinite Scale only with ownCloud Web, but of course, the existing mobile clients and the desktop client will work just like with ownCloud server when connected to Infinite Scale if used in their most recent versions:
```

Comment:
I hope final Desktop app 3.0 and iOS app 12.0 are available at the time of oCIS GA. 🤞

Reply:
> I hope final Desktop app 3.0 and iOS app 12.0 are available at the time of oCIS GA. crossed_fingers

Ok, we'll keep it vaguer and our fingers crossed!